pkg/services/action.go

```diff
@@ -13,18 +13,13 @@ import (
 	"github.com/cmsgov/easi-app/pkg/models"
 )
 
+// ActionExecuter is a function that can execute an action
+type ActionExecuter func(context.Context, *models.SystemIntake, *models.Action) error
```

Comment:
This needs a better name

Reply:
I agree that naming is _haaaard_... 
But I really don't find any fault with this choice. 👍